huntflow_api_client/api.py

```diff
@@ -111,6 +111,7 @@ async def _request(  # type: ignore[no-untyped-def]
                 timeout=timeout,
             )
             await self._raise_token_expired(response)
+            response.raise_for_status()
```

Comment:
It's a bad idea to use default `response.raise_for_status()`

Reply:
Why? What are the alternatives?